### PR TITLE
[1.12] Journald 12

### PR DESCRIPTION
**Pull request by haircommander**
Allowed for user to flag --log-to-journal to log container output to journald as well as kubernetes logs (in conmon).

Backport to release 1.12

### CODE REVIEW

File: docs/crio.8.md [hunk 1]
```diff
@@ -115,6 +116,13 @@ If `hooks_dir` is unset, CRI-O will currently default to `/usr/share/containers/
 
 **--log-size-max**="": Maximum log size in bytes for a container (default: -1 (no limit)). If it is positive, it must be >= 8192 (to match/exceed conmon read buffer).
 
+<<<<<<< HEAD
```
Comment (TomSweeneyRedHat) [inline docs/crio.8.md]: Looks like you've merge bits left behind here.

Comment (haircommander) [reply docs/crio.8.md]: good eye!

**Comment by haircommander** [comment]
@mrunalp happy tests PTAL

**Comment by giuseppe** [comment]
LGTM

**Comment by haircommander** [comment]
needs /approve :)

**Pull request merged**